collections/Lat_th/MS_Lat_th_e_22.xml

```diff
@@ -6,17 +6,12 @@
             <title>MS. Lat. th. e. 22</title>
             <title type="collection">MSS. Lat. th. (Latin theology)</title>
             <respStmt>
-               <resp>Summary description</resp>
-               <persName>Elizabeth Solopova</persName><persName>Matthew Holford</persName>
-            </respStmt>
-         </titleStmt>
-         <editionStmt>
-            <edition>TEI P5</edition>
-            <respStmt>
-               <resp>Conversion to TEI based on earlier work for the ENRICH project (enrich@oucs.ox.ac.uk; http://tei.oucs.ox.ac.uk/ENRICH/)</resp>
+               <resp>Description</resp>
+               <persName>Elizabeth Solopova</persName>
                <persName>Matthew Holford</persName>
```

Comment:
Elizabeth's name and mine should be replaced by "Bodleian library staff", I think, judging from the source element below. My mistake originally